### PR TITLE
Show auction skills on index and show pages

### PR DESCRIPTION
**Pull request by jessieay**
* Closes https://github.com/18F/micropurchase/issues/835

### CODE REVIEW

**Comment by jessieay** [comment]
@mtorres253 do we still want these to be under a "Languages" header per the mocks or should we use the more general term "skills" ?

File: app/models/skill.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class Skill < ActiveRecord::Base
   has_and_belongs_to_many :auctions
+  validates :name, presence: true, uniqueness: true
```
Comment (harrisj) [inline app/models/skill.rb]: Oh yes, that is important

File: app/view_models/auction_list_item.rb [hunk 2]
```diff
@@ -38,6 +38,10 @@ def eligibility_label
     end
   end
 
+  def skills
+    auction.skills.map(&:name).join(', ')
```
Comment (harrisj) [inline app/view_models/auction_list_item.rb]: Should these be sorted (we could put a default order on the association even if you'd prefer)?

File: app/views/auctions/_auction.html.erb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 <div class="js-auction-view">
   <div class="auction-body">
     <div class="auction-description">
+      <h2>Skills</h2>
```
Comment (harrisj) [inline app/views/auctions/_auction.html.erb]: Do we want this header to be displayed if we have not assigned skills to an auction?

File: app/views/auctions/_list_item.html.erb [hunk 4]
```diff
@@ -37,6 +37,13 @@
               <%= auction.eligibility_label %>
             </div>
           </li>
+
+          <li class="issue-details-list-item">
```
Comment (harrisj) [inline app/views/auctions/_list_item.html.erb]: Do we want this to be displayed if we have not assigned skills to an auction?

File: app/view_models/auction_list_item.rb (hunk 2 above)
Comment (jessieay) [inline app/view_models/auction_list_item.rb]: sure, let's do alpha. wil update

Comment (jessieay) [inline app/view_models/auction_list_item.rb]: added method that sorts skills by name and returns names at 293de95

File: app/views/auctions/_auction.html.erb (hunk 3 above)
Comment (jessieay) [inline app/views/auctions/_auction.html.erb]: I think so...wasn't specified in story, but I imagine we will always have some skills for an auction.

**Comment by harrisj** [comment]
Okay, looks good to me. Merging.

**Pull request merged**